src/clj/rems/application/commands.clj

```diff
@@ -509,17 +526,14 @@
       created-event-or-errors
       (let [created-event (:event created-event-or-errors)
             old-app-id (:application/id application)
-            new-app-id (:application/id created-event)]
+            new-app-id (:application/id created-event)
+            values (copied-field-values application new-app-id injections)]
```

Comment:
Hmm should this rather be named `copy-field-values!`

Reply:
Maybe it would also make sense to get the `copy-attachment!` injection out so that it's obvious which fn the `new-app-id` is for. It would also make it much clearer what injection is needed here and why.